docs/jina-ai-cloud/login.md

```diff
@@ -81,14 +81,14 @@ import hubble
 hubble.login()
 ```
 
-#### Login from Jupyter notebook/Google Colab
```

Comment:
I think that if login works the same as in Python, there is no need to have an extra section

Reply:
I'm okay with removing it